tests/python/pants_test/binaries/test_binary_util.py

```diff
@@ -339,3 +341,21 @@ def test_disallowing_external_urls(self):
       "external_url_generator=ExternalUrlGenerator(<example __str__()>), archiver=None): "
       "--binaries-baseurls is empty.")
     self.assertIn(expected_msg, the_raised_exception_message)
+
+  def test_select_argv(self):
+    """Test invoking binary_util.py as a standalone script."""
+    with temporary_dir() as tmp_dir:
+      config_file_loc = os.path.join(tmp_dir, 'pants.ini')
+      safe_file_dump(config_file_loc, """\
+[GLOBAL]
+allow_external_binary_tool_downloads: True
+pants_bootstrapdir: {}
+""".format(tmp_dir), binary_mode=False)
+      expected_output_glob = os.path.join(
+        tmp_dir, 'bin', 'cmake', '*', '*', '3.9.5', 'cmake')
+      with environment_as(PANTS_PANTS_CONFIG_FILES='[{!r}]'.format(config_file_loc)):
```

Comment:
Not important, but I believe you can shorten this to `PANTS_CONFIG_FILES`. I recall special-casing the env vars for global options starting with "pants-", to eliminate stutter.

Reply:
Indeed you can (this is visible by just grepping the string in the repo) -- and it's always good to set a good example, so I've made the change!